hadoop-hdds/framework/src/main/java/org/apache/hadoop/ozone/audit/AuditLoggerType.java

```diff
@@ -25,7 +25,8 @@ public enum AuditLoggerType {
   OMLOGGER("OMAudit"),
   SCMLOGGER("SCMAudit"),
   S3GLOGGER("S3GAudit"),
-  OMSYSTEMLOGGER("OMSystemAudit");
+  OMSYSTEMLOGGER("OMSystemAudit"),
+  OMDELETIONLOGGER("OMDeletionAudit");
```

Comment:
IMO, we should not introduce another Audit log for deletion, already System audit is created for all internal request and major event to be audited.